articles/sql.md

```diff
@@ -0,0 +1,86 @@
+---
+authorName: Ognjen Blagojević
+authorGithubUsername: ognjen-blagojevic-zuehlke
+issue: 85
+title: Do something about that slow SQL query
+---
+# {{page.title}}
+
+Solving the problem of poor database performance is the topic on which the
```

Comment:
Consider a catchier intro. What about:
"The database often causes performance problems in applications. Here's what you can do about it!"

Reply:
Added more poetic intro. :)